### PR TITLE
Fillwidth: Prevents loading two versions of same images; gene page fixes

### PR DESCRIPTION
**Pull request by dzucconi**
To use `FillwidthItem` you should only need to pass it an `imageHeight` everything else is available from the schema. So this change makes `imageHeight` required then removes all the rest of the sizing props and cleans up the call sites. There should be no visual difference with this change.

In making this change I also noticed that there was a hack around attempting to load the correct size image. This was causing 1x devices to load both the 2x and 1x versions — of every thumbnail on the page. The savings should be significant.

This also fixes some fall out that was noticed on gene desktop pages by wrapping the contents in the theme and breakpoints providers as well as removing the old-style `Fillwidth` in favor of a `Carousel` + `FillwidthItem`.

### CODE REVIEW

**Comment by dzucconi** [comment]
To put some numbers on this — comparing local-staging vs staging on a 1x screen, filtering image requests:

```
/artist/andy-warhol

local

41 requests
693 kB transferred

staging

52 requests
1.3 MB transferred
```

```
/artist/r-h-quaytman

local

15 requests
198 kB transferred

staging
26 requests
730 kB transferred
```

This only affects images which are above the fold, so that limits things.

**Comment by dzucconi** [comment]
Another win is that you don't see the images flash as the JS takes over and starts reloading them.

File: src/desktop/apps/gene/client.js [hunk 1]
```diff
@@ -62,7 +63,11 @@ export const setupGenePage = () => {
       user={user ? user.toJSON() : null}
       mediator={mediator}
     >
-      <Contents {...options} onStateChange={onStateChange} />
+      <Theme>
```
Comment (damassi) [inline src/desktop/apps/gene/client.js:66]: Might be better to wrap this in `<Boot>`?

Comment (dzucconi) [reply src/desktop/apps/gene/client.js:66]: Ah yeah. Will do.

**Pull request merged**